### ISSUE TITLE
Folder listing and nav link behavior

### ISSUE THREAD

**Issue by skleinfeldt** [jazkarta/jazkarta.com#85]
Click on Services, then Plone Websites, OR About then Team. Service and About are folders set to Summary View. Plone Websites and Team are folders with a default page set (the cover). The link to them from their parent folder does not go to the default page, it goes to /services/plone-websites/view or /about/team/view.

I don't think I'll be using any folder listings on the site, so this probably has very low priority. But it's weird, and if it is a Plone bug it may be a somewhat serious one.

**Comment by skleinfeldt** [comment]
I take it back, this is a problem because it also affects all the nav links!

**Comment by alecpm** [comment]
Ugh, I wonder what happened to the site wide typesUseViewInListings property.  Will take a look.

**Comment by cewing** [comment]
In the most recent release, there was some talk about having gotten the last of portal_properties ported to the configuration registry.  I'm guessing this might be related?

**Comment by alecpm** [comment]
Something had erroneously added "Folder" to the list of types to use /view on in the registry (and removed Image and File).  There's nothing in the current build that would do that (there are no profiles in the build currently that sets those values, and the schema defaults are Image and File only).

It looks like there was some bad Plone migration code that got removed that borked this field by migrating the wrong data from portal_properties, it's already been fixed, but only after we were smacked by it.  I've added the correct types to the profile and changed the values manually on beta.
